### PR TITLE
Merging team debug branch to finalize our master.

### PR DESCRIPTION
**Pull request by sharonyang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,41 +13,24 @@ Download the "Community" version here:
...
-```
-
-- TODO(team): More to come.
-
-** Note that zlib1g-dev needs to be installed prior to Python installation.
-If the order is reversed, the Python structure might need a re-"make install".
+be installed (via pip for Python 2).
 
+- jsonpickle
+- numpy
```
Comment (yubo56) [inline README.md:23]: matplotlib

File: README.md [hunk 2]
```diff
@@ -13,41 +13,24 @@ Download the "Community" version here:
...
-TODO(team): Modify this once the other wrapper programs are completed.
-We can use setup.py later.
-
-Currently, to use our classes, do:
+Run initializer.py (or read the file) to see example usage. One method of
+example usage is included below:
 ```
-python
-from network_simulator import *
+python2 initializer.py -v INFO -f stdout data/test_case_0_fast.json
```
Comment (yubo56) [inline README.md]: -L <links> in usage as well?

File: network_simulator/common.py [hunk 3]
```diff
@@ -58,9 +59,21 @@
...
 """
-GRAPH_WINDOW_SIZE = 0.05
\ No newline at end of file
+GRAPH_WINDOW_SIZE = 0.10
+
+"""
+If non-None, this is a list of the Links for which we want to see stats. This
+is used to reduce visual clutter (especially in Test Cases 1-2).
+"""
+LINKS_TO_CALC_STATS = None
```
Comment (yubo56) [inline network_simulator/common.py]: Take this out since we are using print_links now

File: README.md (hunk 1 above)
Comment (sharonyang) [inline README.md:23]: Done.

File: README.md (hunk 2 above)
Comment (sharonyang) [inline README.md]: Done.

File: network_simulator/common.py (hunk 3 above)
Comment (sharonyang) [inline network_simulator/common.py]: Done.

**Comment by dluo11** [comment]
It looks good to me

**Pull request merged**